hdfs3/tests/test_hdfs3.py

```diff
@@ -27,6 +28,7 @@ def hdfs():
 
     if hdfs.exists('/tmp/test'):
         hdfs.rm('/tmp/test')
+    hdfs.disconnect()
```

Comment:
Here is more info http://stackoverflow.com/questions/18971451/how-many-times-can-del-be-called-per-object-in-python see Tim Peters answer.